Android NDK build error with `AMediaCodecOnAsyncNotifyCallback`

After bumping up the android api level from 24 to 28, following compile error occurs.
Error log:
```
 ...
 [arm64-v8a] Compile        : nnstreamer <= gstamcsrc.c                                                                                                                                                           

  In file included from /home/yongjoo/nnstreamer/ext/nnstreamer/android_source/gstamcsrc.c:74: 
home/yongjoo/android/tools/sdk/ndk/20.1.5948944/toolchains/llvm/prebuilt/linux-86_64/sysroot/usr/include/media/NdkMediaCodec.h:388:9: error: must use 'struct' tag to refer to type 'AMediaCodecOnAsyncNotifyCallback' 
 AMediaCodecOnAsyncNotifyCallback callback,                                                                                                                                                               
          ^ 
 1 error generated.
  make: *** [/home/yongjoo/nnstreamer/build_android_all/api/build/intermediates/ndkBuild/debug/obj/local/arm64-v8a/objs-debug/nnstreamer//home/yongjoo/nnstreamer/ext/nnstreamer/android_source/gstamcsrc.o] Error 1                                                                                                                                                                                                                 
...
```

Should update the NDK source code after the master branch:
https://android.googlesource.com/platform/frameworks/av/+/master/media/ndk/include/media/NdkMediaCodec.h#117

--- a/api/android/build-android-lib.sh
+++ b/api/android/build-android-lib.sh
@@ -205,6 +205,11 @@ fi
 echo "Android SDK: $android_sdk_dir"
 echo "Android NDK: $android_ndk_dir"
 
+echo "Patching NDK source"
+# See: https://github.com/nnstreamer/nnstreamer/issues/2899
+
+sed -z -i "s|struct AMediaCodecOnAsyncNotifyCallback {\n      AMediaCodecOnAsyncInputAvailable  onAsyncInputAvailable;\n      AMediaCodecOnAsyncOutputAvailable onAsyncOutputAvailable;\n      AMediaCodecOnAsyncFormatChanged   onAsyncFormatChanged;\n      AMediaCodecOnAsyncError           onAsyncError;\n};|typedef struct AMediaCodecOnAsyncNotifyCallback {\n      AMediaCodecOnAsyncInputAvailable  onAsyncInputAvailable;\n      AMediaCodecOnAsyncOutputAvailable onAsyncOutputAvailable;\n      AMediaCodecOnAsyncFormatChanged   onAsyncFormatChanged;\n      AMediaCodecOnAsyncError           onAsyncError;\n} AMediaCodecOnAsyncNotifyCallback;|" $android_ndk_dir/toolchains/llvm/prebuilt/*/sysroot/usr/include/media/NdkMediaCodec.h
+
 # GStreamer prebuilt libraries for Android
 # Download from https://gstreamer.freedesktop.org/data/pkg/android/
 if [[ -z "$gstreamer_dir" ]]; then

